internal/util/defaults_upstream.go

```diff
@@ -0,0 +1,9 @@
+//go:build upstream
+
+package util
+
+const (
```

Comment:
Is the idea that downstream would have its own defaults?

Reply:
Yep! downstream, we can add a `defaults_downstream.go` file (which should never conflict because it has `_downstream` in the name), and then override `GO_BUILD_TAGS` to `downstream` when we build.

Reply:
This build tag mechanism is also the way we can manage differences in code/functionality between upstream and downstream as well.